scrum/tools/sprint_math.py

```diff
@@ -265,7 +299,7 @@ def calc_current_sprint(self):
         self.sp_retro_total = self.sp_retro_completed + self.sp_retro_leftover
 
         # Calculate the next Sprint
-        self.calc_planned_next_sprint()
+        return self.calc_planned_next_sprint()
```

Comment:
given the name and description of this function (calc_current_sprint), I would not expect it to return anything. Where is calc_planned_next_sprint() used? It's probably better called directly or given a dedicated get_ method

Reply:
Pylint, again. Fixed now though